docs/contributors_guide/maintenance/Forked_Puppet_Module_Release_Procedures.rst

```diff
@@ -0,0 +1,77 @@
+Forked Puppet Module Release Procedures
+=======================================
+
+This section will describe the release procedures for projects
+for which SIMP is not the owner.
+
+.. Important::
+
+   If SIMP has made modifications to a project that have
+   not been accepted by the owner **and** are needed by SIMP,
+   the SIMP Team must create a SIMP-owned fork of the project.
```

Comment:
@op-ct I thought the text was clear that the only reason to change the name is *IF* we have made modifications that have not been accepted upstream.